src/games/strategy/engine/data/ChangeFactory.java

```diff
@@ -69,10 +69,6 @@ public static Change changeOwner(final Unit unit, final PlayerID owner, final Te
     return new PlayerOwnerChange(list, owner, location);
   }
 
-  public static Change changeUnitProduction(final Territory terr, final int value) {
-    return new ChangeUnitProduction(terr, value);
```

Comment:
If the class is no longer being used, please delete the class as well (ChangeUnitProduction, etc)